server/routes/confirmedOnReport/confirmedOnReport.ts

```diff
@@ -19,22 +19,10 @@ export default class ConfirmedOnReportRoutes {
     )
 
     return res.render(`pages/confirmedOnReport`, {
-      adjudicationNumber,
```

Comment:
If all this is deleted then can we not get rid of lots of the code above, maybe even delete `reportedAdjudicationsService.getConfirmationDetails`?